schemas/dab.draft.schema.json

```diff
@@ -747,7 +747,7 @@
                   "properties": {
                     "type": {
                       "type": "string",
-                      "enum": ["table", "view", "stored-procedure"],
+                      "enum": [ "table", "view", "stored-procedure" ],
```

Comment:
`"description": "Entities that will be exposed via REST and/or GraphQL",`

      Should mention MCP too in the description above.